src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -73,6 +73,14 @@ using ::open;
 
 namespace {
 
+bool BoolFromEnvVar(const char* env_var, bool default_value) {
+  const char* value = getenv(env_var);
+  if (value) {
+    return std::string("YES") == ToUpper(value);
```

Comment:
Could simplify reading this to `return ToUpper(value) == "YES";`

Reply:
Is 'yes' the only case we handle? 

https://developer.apple.com/documentation/foundation/nsstring/1409420-boolvalue?language=objc as an example

Reply:
> Could simplify reading this to `return ToUpper(value) == "YES";`

It still has to build the `std::string`, so I'm not sure it really counts as simplier.

> Is 'yes' the only case we handle?
> 
> https://developer.apple.com/documentation/foundation/nsstring/1409420-boolvalue?language=objc as an example

We don't need to match everything, just going for the basics.